apps/emqx_exhook/src/emqx_exhook.appup.src

```diff
@@ -1,13 +1,13 @@
-%% -*-: erlang -*-
+%% -*- mode: erlang -*-
 {VSN,
  [
-    {<<"4.3.[0-3]">>, [
+    {<<"4.3.[0-4]">>, [
```

Comment:
Let's use the `update` instruction instead of restarting the application directly.

That will cause a crash error if the emqx_exhook_mngr process is created by exhook Module